lib/tasks/demo-build.js

```diff
@@ -88,29 +89,62 @@ function buildJs(buildConfig) {
 	}
 }
 
+function loadRemoteDemoData(url) {
+	const genericErrorMessage = 'Could not load remote demo data.';
+	return request.get({
+		uri: url,
+		json: true
+	})
+	.then(data => {
+		if (typeof data === 'object') {
+			return data;
+		}
+		throw new Error(`${genericErrorMessage} ${url} did not provide valid JSON.`);
+	}).catch((err) => {
+		if (err.name === 'RequestError') {
+			err.message = `${genericErrorMessage} ${url} does not appear to be valid.`;
+		}
+		if (err.name === 'StatusCodeError') {
+			  err.message = `${genericErrorMessage} ${url} returned a ${err.statusCode} status code.`;
+		}
+		const e = new Error(err.message);
+		e.stack = '';
+		throw e;
+	});
+}
+
+function loadLocalDemoData(dataPath) {
+	return fileExists(dataPath)
+		.then(exists => {
+			if (exists) {
```

Comment:
Do we want to throw an error if the local file doesn't exist? I know it's not in there currently but seems like it should if we're modifying this code anyway.